### PR TITLE
Add authentication after a user registered

### PR DESCRIPTION
**Pull request by andrew-kuzmin**
@ybatsiun check if the AuthenticationResponse is enough 
@JackKarichkovskiy I decided simply authenticate a user when he would register

### CODE REVIEW

**Comment by ybatsiun** [comment]
Ok, will check it

**Comment by ybatsiun** [comment]
All good from my side @andrew-kuzmin @JackKarichkovskiy

File: server/src/main/java/com/phoenixoft/teambalanceapp/controller/AuthenticationController.java [hunk 1]
```diff
@@ -33,8 +32,10 @@ public AuthenticationResponse createAuthenticationToken(@RequestBody Authenticat
     }
 
     @PostMapping(path = "/register")
-    public UserResponseDto registerNewUser(@RequestBody @Valid UserRegistrationRequestDto dto) {
-        return DtoConverter.convertUser(userService.registerNewUser(dto));
+    public AuthenticationResponse registerNewUser(@RequestBody @Valid UserRegistrationRequestDto dto) {
+        userService.registerNewUser(dto);
+        AuthenticationRequest authenticationRequest = new AuthenticationRequest(dto.getEmail(), dto.getPassword());
+        return authenticationService.authenticateUser(authenticationRequest);
```
Comment (JackKarichkovskiy) [inline server/src/main/java/com/phoenixoft/teambalanceapp/controller/AuthenticationController.java:38]: We can just call `jwtTokenUtil.generateToken(user)` to generate JWT for newly created user. What do you think?

Comment (andrew-kuzmin) [reply server/src/main/java/com/phoenixoft/teambalanceapp/controller/AuthenticationController.java]: Yes, you are right

**Pull request merged**